source/_guides/appstore-publishing/preparing-a-submission.md

```diff
@@ -27,9 +27,9 @@ privately. The appstore is built into the official mobile apps and means that
 every new app can be found and also featured for increased exposure and
 publicity.
 
-> Note: An app can only be published privately while it is not already published
-> publicly. If an app is already public, it must be unpublished before it can be
-> made private.
+# > Note: An app can only be published privately while it is not already published
+# > publicly. If an app is already public, it must be unpublished before it can be
+# > made private.
```

Comment:
Did you meant to comment this section out? Because they're just headings right now:

<img width="659" height="638" alt="image" src="https://github.com/user-attachments/assets/90d8eab7-2515-45c3-86de-feeb2403b261" />

Reply:
Whoops. Yes I did